src/main/java/edu/ie3/datamodel/utils/ContainerNodeUpdateUtil.java

```diff
@@ -0,0 +1,639 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.utils;
+
+import edu.ie3.datamodel.models.input.MeasurementUnitInput;
+import edu.ie3.datamodel.models.input.NodeInput;
+import edu.ie3.datamodel.models.input.connector.*;
+import edu.ie3.datamodel.models.input.container.*;
+import edu.ie3.datamodel.models.input.graphics.LineGraphicInput;
+import edu.ie3.datamodel.models.input.graphics.NodeGraphicInput;
+import edu.ie3.datamodel.models.input.system.*;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.locationtech.jts.geom.Point;
+
+public class ContainerNodeUpdateUtil {
```

Comment:
I had it like this before but this resulted in a very bloat/big `ContainerUtils` which I thought might be harder to maintain and use because of the number of methods in the `ContainerUtils` then ... I'm open to discuss other ways to deal with this, but I would prefer having class files (in particular utility class files) with a number of lines below 1k